src/test/java/uk/gov/gchq/koryphe/impl/binaryoperator/DeduplicateConcatTest.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2017 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.gchq.koryphe.impl.binaryoperator;
+
+import org.junit.Test;
+import uk.gov.gchq.koryphe.binaryoperator.BinaryOperatorTest;
+import uk.gov.gchq.koryphe.util.JsonSerialiser;
+import java.io.IOException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class DeduplicateConcatTest extends BinaryOperatorTest {
+
+    @Test
+    public void shouldRemoveDuplicate() {
+        // Given
+        final StringDeduplicateConcat sdc = new StringDeduplicateConcat();
+
+        // When
+        String output = sdc._apply("test,string", "test,success");
+
+        // Then
+        assertEquals("test,string,success", output);
+    }
+
+    @Test
+    public void shouldHandleDefinedDelimiter() {
+        // Given
+        final StringDeduplicateConcat sdc = new StringDeduplicateConcat();
+        sdc.setSeparator(";");
+
+        // When
+        String output = sdc._apply("test;string", "with;delimiter");
+
+        // Then
+        assertEquals("test;string;with;delimiter", output);
+    }
+
+    @Override
+    protected StringDeduplicateConcat getInstance() {
+        return new StringDeduplicateConcat();
+    }
+
+    @Override
+    protected Class<StringDeduplicateConcat> getFunctionClass() {
+        return StringDeduplicateConcat.class;
+    }
+
+    @Test
+    public void shouldJsonSerialiseAndDeserialise() throws IOException {
+        // Given
+        final StringDeduplicateConcat sdc = new StringDeduplicateConcat();
+
+        // When 1
+        final String json = JsonSerialiser.serialise(sdc);
+
+        // Then 1
+        JsonSerialiser.assertEquals(String.format("{%n" +
+                "  \"class\" : \"uk.gov.gchq.koryphe.impl.binaryoperator.StringDeduplicateConcat\",%n" +
+                "  \"separator\" : \",\"%n" +
+                "}"), json);
+
+        // When 2
+        final StringDeduplicateConcat deserialisedOperator =
+                JsonSerialiser.deserialise(json, getFunctionClass());
+
+        // Then 2
+        assertNotNull(deserialisedOperator);
```

Comment:
Please could you also check the separator is serialised to and from json correctly. I.e set the serialiser to be ";" check the json, then deserialise it back into the StringDeduplicateConcat and check the separator is correct.